.github/workflows/R-CMD-check.yaml

```diff
@@ -71,6 +71,10 @@ jobs:
           remotes::install_cran("lintr")
         shell: Rscript {0}
 
+      - name: Install package
+        if: runner.os == 'macOS' || runner.os == 'Linux'
+        run: R CMD INSTALL .
+
```

Comment:
Hmmm, I am unsure if this is the right move: installing the package before checking it. Still, all tests pass and you must have put it there for a good reason, so I accept :+1:

Reply:
You are right. We just need to do it before we run lint. I will move it closer to lint. If this is not there, the build will fail on GHA, while giving no error locally (locally we always run R CMD INSTALL).